Homework/ViewController.swift

```diff
@@ -1,19 +1,191 @@
-//
-//  ViewController.swift
-//  Homework
-//
-//  Created by Egor on 04.09.2022.
-//
-
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    //Создание кнопки сложения
+    var additionButton = UIButton()
+    //Создание кнопки угадывания числа
+    var numberGuessingButton = UIButton()
+    //Создание лэйбла, в котором будет отображаться ФИО пользователя
+    var labelFullName = UILabel()
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        //Настройка лэйбла
+        labelFullName = UILabel(frame: CGRect(x: 50, y: 300, width: 320, height: 50))
+        labelFullName.textAlignment = .center
+        labelFullName.text = "Привет, "
+        labelFullName.font = labelFullName.font.withSize(25)
+        labelFullName.backgroundColor = .gray
+        labelFullName.isHidden = true
+        labelFullName.textColor = .black
+        self.view.addSubview(labelFullName)
```

Comment:
```suggestion
        view.addSubview(labelFullName)
```

селф пишем только в замыкания и клоужерах, тут не надо